gitian-building.md

```diff
@@ -76,6 +72,18 @@ git remote add $NAME git@github.com:$NAME/gitian.sigs.git
 
 Where `satoshi` is your GitHub name.
 
+macOS code signing
+------------------
+In order to sign builds for macOS, you need to download the free SDK and extract a file.
+The steps are described [here](https://github.com/bitcoin/bitcoin/blob/master/contrib/macdeploy/README.md#sdk-extraction).
```

Comment:
```suggestion
The steps are described [here](https://github.com/bitcoin/bitcoin/blob/master/contrib/macdeploy/README.md#sdk-extraction) for `Xcode-11.3.1-11C505-extracted-SDK-with-libcxx-headers.tar.gz`.
The steps are described [here](https://github.com/bitcoin-core/docs/blob/a706cc6afde0c8999fb0b8e3356522a7f8e2ad8f/gitian-building/gitian-building-mac-os-sdk.md) for `MacOSX10.11.sdk.tar.gz`.
```

Reply:
Updated.

But a new list of the explicitly mentioned extracted SDKs misses `MacOSX10.14.sdk.tar.gz`.